File: C#/Lv_0/순서 바꾸기.cs
using System;

public class Solution {
    public int[] solution(int[] num_list, int n) {
        int[] answer = new int[num_list.Length];
        int j=0;
        for(int i=n; i<num_list.Length; i++, j++) answer[j] = num_list[i];
        for(int i=0; i<n; i++, j++) answer[j] = num_list[i];
        return answer;
    }
}

File: JavaScript/Lv_0/순서 바꾸기.js
function solution(num_list, n) {
    var answer = [];
    for(let i=n; i<num_list.length; i++) answer.push(num_list[i]);
    for(let i=0; i<n; i++) answer.push(num_list[i]);
    return answer;
}

// 간단한 방법, splice를 활용!
// function solution(num_list, n) {
//     num_list.push(...num_list.splice(0, n));
//     return num_list
// }